website/static/js/config.js

```diff
@@ -0,0 +1,153 @@
+  cookieName: 'substrateDevHubCookie',
+
+  // You can also set a custom expiration time for the Klaro cookie.
+  // By default, it will expire after 120 days.
+  cookieExpiresAfterDays: 120,
+
+  // You can customize the name of the cookie that Klaro will use to
+  // store user consent. If undefined, Klaro will use 'klaro'.
+
+  // Put a link to your privacy policy here (relative or absolute).
+  privacyPolicy: 'https://www.parity.io/privacy/',
+
+  // Defines the default state for applications (true=enabled by default).
+  default: true,
+
+  // If "mustConsent" is set to true, Klaro will directly display the consent
+  // manager modal and not allow the user to close it before having actively
+  // consented or declines the use of third-party apps.
+  mustConsent: false,
+
+  // You can define the UI language directly here. If undefined, Klaro will
+  // use the value given in the global "lang" variable. If that does
+  // not exist, it will use the value given in the "lang" attribute of your
+  // HTML tag. If that also doesn't exist, it will use 'en'.
+  //lang: 'en',
+
+  // You can overwrite existing translations and add translations for your
+  // app descriptions and purposes. See `src/translations.yml` for a full
+  // list of translations that can be overwritten:
+  // https://github.com/DPKit/klaro/blob/master/src/translations.yml
+
+  // Example config that shows how to overwrite translations:
+  // https://github.com/DPKit/klaro/blob/master/src/configs/i18n.js
+  translations: {
+    // If you erase the "consentModal" translations, Klaro will use the
+    // defaults as defined in translations.yml
+    de: {
+      consentModal: {
+        description:
+          'Hier können Sie einsehen und anpassen, welche Information wir über Sie sammeln. Einträge die als "Beispiel" gekennzeichnet sind dienen lediglich zu Demonstrationszwecken und werden nicht wirklich verwendet.',
+      },
+      googleAnalytics: {
+        description: 'Sammeln von Besucherstatistiken',
+      },
+      cloudflare: {
+        description: 'Schutz gegen DDoS-Angriffe',
+      },
+      purposes: {
+        analytics: 'Besucher-Statistiken',
+        security: 'Sicherheit',
+      },
+    },
+    en: {
+      consentModal: {
+        description:
+          'We use necessary cookies to make our site work. We\'d also like to set optional tracking mechanisms to help us improve it by collecting and reporting information on how you use it. Here you can customise the information we collect about you by giving or revoking your consent to our use of optional tracking mechanisms.',
+      },
+      googleAnalytics: {
+        description: 'Collection of information about how visitors use our website',
+      },
+      cloudflare: {
+        description: 'Protection against DDoS attacks',
+      },
+      purposes: {
+        analytics: 'analytics and improvement of our sites',
+        security: 'security',
+      },
+    },
+    tr: {
```

Comment:
It seems our second most popular language is Chinese. Should we have that translation here? Also how does Klaro detect the language to use. Will it automatically show the consent box in the correct language if the user selects a different language for devhub content?

Reply:
+1, but I don't think should be a blocker to be merged. If we can address it in a follow up PR, that is fine too

Reply:
Klaro doesn't support Chinese yet, https://github.com/KIProtect/klaro/blob/master/src/translations/index.js

Reply:
Klaro fixed in Chinese page after update it to the latest.
Later PR will add Chinese support.